python/pyspark/sql/window.py

```diff
@@ -89,6 +89,9 @@ def rowsBetween(start, end):
         Creates a :class:`WindowSpec` with the frame boundaries defined,
         from `start` (inclusive) to `end` (inclusive).
 
+        Rows Between cares only about the order of rows, and takes fixed number of
```

Comment:
"Rows Between" -> "rowsBetween"? but I'm also not quite sure what this means. Ordering matters and yes it seems kind of implied that the number is fixed? what's a 'frame'?

Is there Scala doc that can be copied instead?